pkg/kn/commands/broker/describe_test.go

```diff
@@ -25,44 +26,60 @@ import (
 	"gotest.tools/v3/assert/cmp"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
-	v1beta1 "knative.dev/eventing/pkg/apis/eventing/v1"
+	eventingv1 "knative.dev/eventing/pkg/apis/eventing/v1"
 	"knative.dev/pkg/apis"
 	duckv1 "knative.dev/pkg/apis/duck/v1"
 
-	clientv1beta1 "knative.dev/client/pkg/eventing/v1"
+	clientv1 "knative.dev/client/pkg/eventing/v1"
 	"knative.dev/client/pkg/util"
 )
 
 func TestBrokerDescribe(t *testing.T) {
```

Comment:
The new tests work regardless of your fix in `NewBrokerBuilder` (so with the original code without the fix in this PR).